skptesting/enckey

```diff
@@ -0,0 +1 @@
+very secure
```

Comment:
where is this file used? Was it added by accident?

Reply:
ah this was used in the dropped test

Reply:
I pushed the test